src/v2/DevTools/setupTestWrapper.tsx

```diff
@@ -94,11 +106,13 @@ export const setupTestWrapperTL = <T extends OperationType>({
 
     const view = render(<TestRenderer />)
 
-    env.mock.resolveMostRecentOperation(operation => {
-      return MockPayloadGenerator.generate(operation, mockResolvers)
-    })
+    if (!manualEnvControl) {
+      env.mock.resolveMostRecentOperation(operation => {
+        return MockPayloadGenerator.generate(operation, mockResolvers)
+      })
+    }
 
-    return view
+    return { ...view, env }
```

Comment:
Surprised this return change didn't break other tests. I guess this is because RTL uses global state via imports to assert against screen content, versus enzyme. where it returns an instance of the wrapper?